### PR TITLE
rename VAR_CF

### PR DESCRIPTION
**Pull request by spacek531**

### CODE REVIEW

**Review by duncanspumpkin — CHANGES_REQUESTED** [review]
Again doesn't make sense what does seat rotation have to do with brake speed

**Comment by spacek531** [comment]
I have no idea but that's the minigolf shadow realm code. Whatever it's trying to read, it's probably not really rotation.

File: src/openrct2/ride/Vehicle.cpp [hunk 1]
```diff
@@ -8940,7 +8940,7 @@ void Vehicle::UpdateTrackMotionMiniGolfVehicle(Ride* curRide, rct_ride_entry* ri
 
     SetTrackType(tileElement->AsTrack()->GetTrackType());
     SetTrackDirection(direction);
-    var_CF = tileElement->AsTrack()->GetSeatRotation() << 1;
+    brake_speed = tileElement->AsTrack()->GetSeatRotation() << 1;
```
Comment (duncanspumpkin) [inline src/openrct2/ride/Vehicle.cpp:8943]: image
Looks like this was wrong it should have been:
```
brake_speed = tileElement->AsTrack()->GetBrakeBoosterSpeed();
```

**Comment by duncanspumpkin** [comment]
Make my change and then that is all users of CF being correct and related to brake booster speed. Also you will need to bump the network version when you make my change.

File: src/openrct2/ParkFile.cpp [hunk 2]
```diff
@@ -1823,7 +1823,7 @@ namespace OpenRCT2
         cs.ReadWrite(entity.scream_sound_id);
         cs.ReadWrite(entity.TrackSubposition);
         cs.ReadWrite(entity.var_CE);
-        cs.ReadWrite(entity.var_CF);
+        cs.ReadWrite(entity.brake_speed);
```
Comment (ZehMatt) [inline src/openrct2/ParkFile.cpp:1826]: In the future use
```PascalCase``` for public variable names

**Pull request merged**